rules/otp.bzl

```diff
@@ -176,33 +63,35 @@ def _otp_json_impl(ctx):
 
     file = ctx.actions.declare_file("{}.json".format(ctx.attr.name))
     ctx.actions.write(file, json.encode_indent(otp))
-    return DefaultInfo(files = depset([file]))
+    return [DefaultInfo(files = depset([file]))]
 
 otp_json = rule(
     implementation = _otp_json_impl,
     attrs = {
-        # Valid life cycle states can be found in the life cycle state
-        # definition file (default: hw/ip/lc_ctrl/data/lc_ctrl_state.hjson)
-        "lc_state": attr.string(doc = "Life cycle state", default = "RMA"),
-        "lc_count": attr.int(doc = "Life cycle count", default = 8),
+        "seed": attr.string(
+            doc = "Seed to be used for generation of partition randomized values. Can be overridden by the OTP image generation tool.",
+        ),
+        "partitions": attr.string_list(doc = "A list of serialized partitions from otp_partition."),
```

Comment:
I have a docstring at the top of the file that explains the setup. I figured it would make more sense to explain the whole OTP generation flow in one place rather than splitting up the docs between several different functions.